posthog/email.py

```diff
@@ -22,17 +22,19 @@ def inline_css(value: str) -> str:
 
 def is_email_available() -> bool:
     """
-    Returns whether email services are available on this instance
-    (i.e. settings are properly configured)
+    Returns whether email services are available on this instance (i.e. settings are in place).
     """
-    return bool(settings.EMAIL_HOST)
+    return bool(settings.EMAIL_HOST and settings.EMAIL_HOST_USER and settings.EMAIL_HOST_PASSWORD)
```

Comment:
@Twixes wouldn't it be possible for instances to use anonymous (i.e. requiring no auth) email servers? Or should we discourage this?

Reply:
I just thought an authless email server is less likely than users forgetting to set user and password.

Reply:
In the old days everyone had a SMTP server (e.g. [exim4](https://www.exim.org/)) running on their server, which then sent e-mails. You could just `nc localhost 25` and start writing your e-mail headers (`MAIL FROM: `, etc), followed by the "view source" version of the mail.

Later on when e-mail became very complicated to maintain yourself (mainly due to spammers), a practice still existed to have a localhost port 25 simple "authless" server that would just proxy everything to a real SMTP server. This also allows standard unix commands (`sendmail`) to send e-mails easily. I'm not sure how many users still have such a setup. Though since authless e-mail is allowed by the [SMTP spec](https://tools.ietf.org/html/rfc5321), we should support it.

(And I see that it's fixed... so thanks for reading :))